### PR TITLE
Proto file for identity and meta service interfaces

### PR DESCRIPTION
**Pull request by sushanthakumar**
The PR includes the Proto file and generated code for the identity and meta service interfaces.
Volume services interfaces to be updated to this at later stage

### CODE REVIEW

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

File: providerframework/proto/providerservice.proto [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    Type type = 1;
+  }
+  message MetaBackupServiceSupport {
+    enum Type {
+      UNKNOWN = 0;
+      NEED_SNAPSHOT = 1;
+    }
+    Type type = 1;
+  }
+  message VolumeBackupServiceSupport {
```
Comment (AmitRoushan) [inline providerframework/proto/providerservice.proto:48]: IMO, The message currently can be ignored and add back once volume services are defined

**Review by AmitRoushan — CHANGES_REQUESTED** [review]

File: providerframework/proto/providerservice.proto (hunk 1 above)
Comment (sushanthakumar) [reply providerframework/proto/providerservice.proto]: Updated

**Review by AmitRoushan — APPROVED** [review]
LGTM

**Pull request merged**